lib/extension/list.extension.js

```diff
@@ -0,0 +1,40 @@
+'use strict'; // eslint-disable-line strict
+/**
+ *  Class of an extension doing the simple listing
+ */
+class List {
+    /**
+     *  Constructor
+     *  Set the logger and the res
+     *  @param {Object} parameters - The parameters you sent to DBD
+     *  @param {RequestLogger} logger - The logger of the request
+     *  @return {undefined}
+     */
+    constructor(parameters, logger) {
+        this.logger = logger;
+        this.res = [];
+    }
+
+    /**
+     *  Function apply on each element
+     *  Just add it to the array
+     *  @param {Object} elem - The data from the database
+     *  @return {Boolean} - True = continue the stream
+     */
+    filter(elem) {
+        this.res.push(elem);
+        return true;
+    }
+
+    /**
+     *  Function returning the result
+     *  @return {Array} - The listed elements
```

Comment:
We do not know, you can store strings, numbers, objects .., it's an array of elem whatever what elem is.